compiler/generator.c

```diff
@@ -586,7 +586,8 @@ static void generate_do(struct generator * g, struct node * p) {
     if (g->label_used)
         wsetl(g, g->failure_label);
     if (keep_c) {
-        w(g, "~M"); wrestore(g, p, keep_c);
+        w(g, "~M");
+		wrestore(g, p, keep_c);
         w(g, "~N~}");
```

Comment:
This sort of unrelated change isn't helpful either.

Perhaps we should change the code to not have multiple statements per line, but if so that should be done as a separate PR.